spotinst_sdk2/clients/elastigroup/__init__.py

```diff
@@ -1767,59 +1753,57 @@ def get_cost_per_elastigroup(self, group_id, to_date=None, from_date=None):
 
         return formatted_response["response"]["items"]
 
-    def get_elastigroup_activity(self, group_id, start_date):
+    def get_elastigroup_activity(self, group_id: str, start_date: str, end_date: str):
         """
-        Get elastigroup activity
+        Get all activity events for the Elastigroup
 
         # Arguments
         group_id (String): Elastigroup ID
         start_date (String): Date when to start checking
+        end_date (String): Get items on or before this date (ISO 8601)
 
         # Returns
         (Object) : Elastigroup API response 
         """
         query_params = self.build_query_params_with_input(
```

Comment:
Why do we need to call build_query_params_with_input(), it is called internally from send_get(). I guess, we only need to send a dict() here.

Reply:
Done